aws/resource_aws_kms_key.go

```diff
@@ -455,3 +471,20 @@ func resourceAwsKmsKeyDelete(d *schema.ResourceData, meta interface{}) error {
 	d.SetId("")
 	return nil
 }
+
+func retryOnAwsCode(code string, f func() (interface{}, error)) (interface{}, error) {
```

Comment:
I wonder if this is reusable and should live with `awserr.isAWSErr()` ? It may not be worth extracting but just a thought, not a blocker at all